_posts/2016-08-09-collections-app.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Collecting Connections through the Powerbox"
+author: David Renshaw
+authorUrl: https://github.com/dwrensha
+---
+
+Starting work on a project in Sandstorm often means creating many grains, each pertaining to a
+different aspect of your work. In a typical project, you might have a spreadsheet, a chat room, a
+kanban board, and several source code repositories. Sandstorm makes it easy to share these grains
+with your collaborators, but until recently, you would need to share each grain to each collaborator
+separately --- a task that could quickly become tedious. What if you want to share all of the
+project's grains as a single unit?
+
+Now that we have released the
+[Collections app](https://apps.sandstorm.io/app/s3u2xgmqwznz2n3apf30sm3gw1d85y029enw5pymx734cnk5n78h),
+we have a satisfying answer to that question; to share many grains at once, you add them to a collection.
```

Comment:
I think it is nearly relevant to everyone including non-developers, but then I would change the title to accommodate them better, e.g. not talk about "connections". Brainstorming examples:

- Collections: Folders, with a Sandstorm twist

- Sandstorm supports collections, with the help of an app

- Creating & sharing collections all at once

- Creating collections: the first Sandstorm feature reliant on the powerbox

- Collections of grains: How to use them, and how anyone can build their own UI

- Collecting & sharing grains all together

- Collections in Sandstorm: Easy for users, and open to developers